"update server"

khasogi27 opened the pull request.

Commit:
initial commit

Commit:
push on new work

Commit:
remove node_modules

khasogi27 commented:
update review & assignees

ziadyjo commented on server/controllers/productController.js:
```diff
@@ -0,0 +1,116 @@
+      image_url: req.body.image_url,
+      price: req.body.price,
+      stock: req.body.stock,
+      UserId,
```
UserIdnya ga perlu yaa, karena ga perlu ada id pemiliknya

ziadyjo commented on server/controllers/productController.js:
```diff
@@ -0,0 +1,116 @@
+    }
+    Product.create(payload)
+      .then(data => {
+        res.status(200).json({
```
jangan lupa status created itu 201

ziadyjo commented on server/middlewares/authorization.js:
```diff
@@ -0,0 +1,28 @@
+          res.status(404).json({
+            message: 'Product not found'
+          })
+        } else if (data.UserId === req.loggedInUser.id) {
```
disini sebetulnya kalo ga pake relasi antara produk dan user, bisa compare rolenya apakah admin atau bukan

ziadyjo commented on server/middlewares/errorHandler.js:
```diff
@@ -0,0 +1,26 @@
+class ErrorHandler {
```
error handlernya lengkapin lagi yaa

Commit:
1. Add tdd product
2. Add api.doc
3. revisi middlewares
4. revisi db user
5. revisi productController

Commit:
update server

Commit:
deploy heroku

Commit:
deploy heroku

Commit:
update package.json

Commit:
upload link deploy in md

Commit:
update server
